### PR TITLE
Modify ru locale

### PR DESCRIPTION
**Pull request by VovkaGoodwin**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [x] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

Added Russian localization for short days of the week and months

### 📝 Changelog

Added Russian localization for short days of the week and months
| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Added Russian localization for short days of the week and months      |
| 🇨🇳 Chinese |    在一周和几个月的短日子里增加了俄语本地化       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 4cec08d</samp>

Added `shortWeekDays` and `shortMonths` properties to `components/date-picker/locale/ru_RU.ts` to provide abbreviated names for the date picker component in Russian. This is part of a larger pull request that improves localization for more languages.

### <samp>🤖 Generated by Copilot at 4cec08d</samp>

* Add support for more languages to the date picker component by updating the `locale` objects in the corresponding files under `components/date-picker/locale` ([link](https://github.com/ant-design/ant-design/pull/43893/files?diff=unified&w=0#diff-b2a1993fdbfbec78515f461f86e165f115ffc1e091873cc3ad1b97949eb72027R19-R33),                           F27

### CODE REVIEW

**Comment by yoyo837** [comment]
Please update snapshot.

**Comment by VovkaGoodwin** [comment]
> Please update snapshot.

Hi, snapshot has been updated

**Comment by afc163** [comment]
@plandem @addictional Could you help to review this PR?

**Comment by yoyo837** [comment]
Any native `ru_RU` speaking community members please help review this PR.

File: components/date-picker/locale/ru_RU.ts [hunk 1]
```diff
@@ -16,6 +16,21 @@ const locale: PickerLocale = {
...
+      'Апр',
+      'Май',
+      'Июнь',
+      'Июль',
+      'Авг',
+      'Сент',
+      'Окт',
+      'Нояб',
+      'Дек',
+    ],
```
Comment (afc163) [inline components/date-picker/locale/ru_RU.ts:33]: It is better to send PR to dayjs: https://github.com/iamkun/dayjs/blob/a9d7d0398d22ebd4bfc3812ca0134a97606d54d9/src/locale/ru.js#L51-L52

Comment (afc163) [reply components/date-picker/locale/ru_RU.ts:33]: ping

Comment (VovkaGoodwin) [reply components/date-picker/locale/ru_RU.ts:33]: Hi, I checked the ru locale in day js, and I saw that it has all of these words, but when im using the ru locale in antd config provider, I get next: 
image

Comment (afc163) [reply components/date-picker/locale/ru_RU.ts:33]: Could you check https://github.com/ant-design/ant-design/issues/43997#issuecomment-1663552566 ?

Comment (VovkaGoodwin) [reply components/date-picker/locale/ru_RU.ts:33]: lol, i checked and as i understood i still need to import 'dayjs/locale/ru'. I didnt expect this, I thought using the locale from antd was enough.
taking this opportunity, I want to ask a stupid question - what is the difference between imports from 'antd/es/*' and 'antd/lib/*'

**Review by AbdullahWins — APPROVED** [review]
good job!

File: components/date-picker/locale/ru_RU.ts (hunk 1 above)
Comment (afc163) [reply components/date-picker/locale/ru_RU.ts:33]: `antd/es` use `import` and `export` as es module, `antd/lib` are commonjs module.

**Pull request closed without merging**